datamodel/initdb.d/03_dcsa_im_v3_0.sql

```diff
@@ -228,23 +221,45 @@ CREATE TABLE dcsa_ebl_v1_0.iso_equipment_code (
     iso_equipment_type_code_a varchar(2) NULL
 );
 
+DROP TABLE IF EXISTS dcsa_ebl_v1_0.equipment CASCADE;
+CREATE TABLE dcsa_ebl_v1_0.equipment (
+	equipment_reference varchar(15) PRIMARY KEY,	-- The unique identifier for the equipment, which should follow the BIC ISO Container Identification Number where possible. According to ISO 6346, a container identification code consists of a 4-letter prefix and a 7-digit number (composed of a 3-letter owner code, a category identifier, a serial number and a check-digit). If a container does not comply with ISO 6346, it is suggested to follow Recommendation #2 “Container with non-ISO identification” from SMDG.
+	-- Unique code for the different equipment size/type used for transporting commodities. The code is a concatenation of ISO Equipment Size Code and ISO Equipment Type Code A and follows the ISO 6346 standard.
+	-- FIXME: Add FK once we have iso_equipment_code data for our test
+	iso_equipment_code char(4) NULL /* REFERENCES dcsa_ebl_v1_0.iso_equipment_code (iso_equipment_code) */,
```

Comment:
Why is the references commented?

Reply:
Fixed (we did not have a test data entry for it when I added it as a commented reference)